test_schemas/en/test_new_skip_condition_group.json

```diff
@@ -0,0 +1,105 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Skip group",
+    "theme": "default",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "default-section",
+            "groups": [
+                {
+                    "id": "do-you-want-to-skip-group",
+                    "title": "Do you want to skip the next group?",
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "do-you-want-to-skip",
+                            "question": {
+                                "id": "do-you-want-to-skip-question",
+                                "title": "Do you want to skip?",
+                                "type": "General",
```

Comment:
Same here, we need to make it clearer. The title of the group is a question, which is confusing when you get to the summary. We should set the title of the group to just like `Group 1`, `Group 2 (Skippable)` etc so on the summary it is clear you skipped/not skipped a group.

Reply:
Fixed, thanks!